template.distillery/Makefile.db

```diff
@@ -30,8 +30,20 @@ ELIOMDEP          := $(ENV_PSQL) $(ELIOMDEP)
 # should be the same as ENV_PSQL)
 OCSIGENSERVER     := $(OCSIGENSERVER)
 OCSIGENSERVER.OPT := $(OCSIGENSERVER.OPT)
+
 ## ---------------------------------------------------------------------
 
+##----------------------------------------------------------------------
+
+pg_dump = pg_dump
+
+pg_ctl  = $(shell \
+           sh -c "which pg_ctl" || \
+           sh -c "which pg_ctlcluster" || \
+           sh -c "if [ -f pg_ctl ]; then (echo ./pg_ctl) else (echo '') fi")
```

Comment:
For `pg_ctl` and `pg_ctlcluster`, see [this link](http://dba.stackexchange.com/questions/72580/missing-the-pg-ctl-package-in-postgres-9-3-installation) (see also PR #123).
The empty string is used in the conditional line 91 to say there's no pg_ctl binary found.